homeassistant/components/sensor/wunderground.py

```diff
@@ -238,3 +283,18 @@ def update_alerts(self):
             _LOGGER.error("Check WUnderground API %s", err.args)
             self.alerts = None
             raise
+
+    @Throttle(MIN_TIME_BETWEEN_UPDATES_FORECAST)
+    def update_forecast(self):
+        """Get the lastest forecast from WUnderground."""
+        try:
+            result = requests.get(self._build_url(_FORECAST), timeout=10).json()
+            if "error" in result['response']:
+                raise ValueError(result['response']["error"]
+                                 ["description"])
+            else:
+                self.forecast = result["forecast"]['txt_forecast']['forecastday']
```

Comment:
hound found 3 "line too long" violations.  Two of these were in the code prior to my changes.  My change added a third line similar to the other two.  Since the two lines were already in existing code I assumed that the new line wouldn't be an issue.